File: Tutorials/ObjC/Example1.5/Example1.5/SampleLoggingDelegate.m
//
//  SampleLoggingDelegate.m
//  Example1.5
//
//  Created by dev on 7/24/19.
//  Copyright © 2019 Penthera. All rights reserved.
//

#import "SampleLoggingDelegate.h"

@implementation SampleLoggingDelegate


- (void)virtuosoConsoleLogLine:(nonnull NSString*)line atLevel:(kVL_LoggingLevel)level
{
    NSLog( @"virtuosoConsoleLogLine for line: %@", line);
}


- (void)virtuosoEventOccurred:(nonnull VirtuosoBaseEvent*)event
{
    NSLog( @"virtuosoEventOccurred for event: %@", [event jsonData]);
}

@end

File: Tutorials/Swift/Example1.5/Example1.5/SampleLoggingDelegate.swift
//
//  SampleLoggingDelegate.swift
//  Example1.5
//
//  Created by Penthera on 7/24/19.
//  Copyright © 2019 Penthera. All rights reserved.
//

import UIKit

class SampleLoggingDelegate: NSObject,  VirtuosoLoggerDelegate {

    func virtuosoEventOccurred(_ event: VirtuosoBaseEvent) {
        print("virtuosoConsoleLogLine event: \(event.jsonData())")
    }
    
    func virtuosoConsoleLogLine(_ line: String, at level: kVL_LoggingLevel) {
        print("virtuosoConsoleLogLine line: \(line)")
    }
    
}


